src/display/api.js

```diff
@@ -1584,14 +1575,14 @@ class PDFPageProxy {
     this._transport.pageCache[this._pageIndex] = null;
 
     const waitOn = [];
-    for (const [intent, intentState] of this._intentStates) {
+    for (const intentState of this._intentStates.values()) {
       this._abortOperatorList({
         intentState,
         reason: new Error("Page was destroyed."),
         force: true,
       });
 
-      if (intent & RenderingIntentFlag.OPLIST) {
```

Comment:
Unless I'm overlooking something, this was the only (and therefore last) place where the `OPLIST` rendering intent bit flag is actually read. If we make this change, can't we now remove it from the enumeration and the `getRenderingIntent` function?

Reply:
Nope, it's being used in https://github.com/mozilla/pdf.js/blob/a936509b77ae4f7e49dea57cf2e2a0aacd062516/src/core/operator_list.js#L611 which is why we (unfortunately) still need it :-)